samples/sample-terraform/tf/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.0"
+    }
+  }
+  required_version = ">= 0.13.0"
+}
+provider "aws" {
+  assume_role {
+    role_arn     = "arn:aws:iam::${var.TARGET_ACCOUNT_ID}:role/${var.TARGET_ACCOUNT_ROLE}"
```

Comment:
Could you make sure the right AWS partition is referenced in the ARN here?

At the moment, it is locked to `aws`. This prevents the use of `aws-us-gov` and other AWS partitions.

Reply:
added data "aws_partition" "current" {}